src/solidity-typechecker/solidity_tenv.ml

```diff
@@ -62,7 +62,7 @@ let is_visible lookup visibility ~origin ~variable =
   | LStatic (Interface, _) ->
       !for_freeton
   | LStatic (Contract, false) ->
```

Comment:
Are you sure this is always true, or should be conditionned with `for_freeton` ?

Reply:
This one: `is_statically_visible visibility ~library:false`?
That's a mistake, as far as I know it should be `~library:true` even for freeton. I will change it.